src/model-viewer-base.ts

```diff
@@ -21,13 +21,11 @@ import {HAS_INTERSECTION_OBSERVER, HAS_RESIZE_OBSERVER} from './constants.js';
 import {makeTemplate} from './template.js';
 import {$evictionPolicy, CachingGLTFLoader} from './three-components/CachingGLTFLoader.js';
 import {ModelScene} from './three-components/ModelScene.js';
-import {ContextLostEvent, Renderer} from './three-components/Renderer.js';
-import {debounce, deserializeUrl, isDebugMode, resolveDpr} from './utilities.js';
+import {ContextLostEvent, sceneRenderer} from './three-components/Renderer.js';
```

Comment:
I changed it because I kept getting confused by lines like `const renderer = renderer.renderer` Is there a better way to disambiguate?

Reply:
My main issue is that `sceneRenderer` is redundant and doesn't align with the original class (`Renderer`). It's actually a smell that everything accesses the Three.js `WebGLRenderer` as a sub-property of this `Renderer`. It suggests that `Renderer` should now "own" its own `WebGLRenderer` but rather be borrowing it from elsewhere.

As a compromise, would you be willing to rename the `Renderer.renderer` to `Renderer.threeRenderer`? This would add missing context, as well as establish a distinction that will be helpful if we ever decide to look into using alternative renderers in the future.

Reply:
SG.